posts/automation/automation-intro.md

```diff
@@ -5,15 +5,50 @@ menu_order: 1
 post_status: publish
 post_excerpt: If you are interested in cool technologies for DevOps and automation like Ansible, Tekton, or ArgoCD, check out this introduction to our project.
 featured_image: _images/automation.jpg
-author: jstejskal
+author: dkornel
 taxonomy:
     category:
         - automation
 
 ---
 
-## Automation introduction
-This is initial blogpost about automation site of the project
+## What is the automation-hub
+Simple answer is that [automation-hub](https://github.com/skodjob/automation-hub) is a collection of [Ansible](https://www.ansible.com/) scripts, [Kubernetes](https://kubernetes.io/) deployment and configuration files, [Tekton](https://tekton.dev/) pipelines, and [ArgoCD](https://argo-cd.readthedocs.io/en/stable/) applications and application-sets.
+That's quite a very long list of used tools within the repository, and we don't even mention jinja2 templates for example!
+However, there is a good reason for it and we will try to take a closer look on all of them.
 
-* Testing automatic update v1
-* Testing automatic update v2
\ No newline at end of file
+### Kubernetes
+Kubernetes is known as the only reasonable way to orchestrate containerized applications in a large scale.
+It provides scalability, reliability, monitoring and many more features for applications running on top of the Kubernetes cluster.
+In our case we are using enterprise implementation of Kubernetes called [Openshift](https://www.redhat.com/en/technologies/cloud-computing/openshift/container-platform).
+That was just a brief introduction of Kubernetes - it is a platform for running containerized applications.
+We don't want to do any deep dive into Kubernetes in this blog post so if you are interested in it, we suggest to read any other blogs, you will find plenty of them.
+Now let's try to answer the question "Why we use it in scope of our experiments"?
+Answer is simple "We use Kubernetes, because we work as **QA** and the applications, which we put under test, are targeted to Kubernetes."
+
+### Ansible
+Do you know the situations when you have many automation scripts written in `bash` or `powershell` and you start loosing your mind in a bunch of `$` or `sed` expressions?
+We know how frustrating can be debugging shell script or running them on the remote machine.
+Thankfully, there is a tool that can makes your life much easier - `Ansible`.
+Ansible is a declarative way how to write automation scrips in yaml format, which also allows you to use libraries with already implemented steps and tasks that interact with many components, so you do not need re-invent the wheel.
+Because the initial installation path of our environment requires a lot of steps, Ansible it the correct way to deduplicate steps, create a templates in `jinja` and drive execution based on tags.
```

Comment:
```suggestion
Because the initial installation path of our environment requires a lot of steps, Ansible is the correct way to deduplicate steps, create a templates in `jinja` and drive execution based on tags.
```